ODPi-Runtime.md

```diff
@@ -143,16 +145,19 @@ The following environment variables are noted by this spec:
 | **[HADOOP_EC3]** | HADOOP_COMMON_DIR    | Relative Dir  | Apache Hadoop common jars |
 | **[HADOOP_EC4]** | HADOOP_COMMON_LIB_JARS_DIR | Relative Dir | Apache Hadoop common jar dependencies |
 | **[HADOOP_EC5]** | HADOOP_CONF_DIR | Absolute Dir | Location of Apache Hadoop configuration files |
+| **[HADOOP_EC6]** | HADOOP_HOME | Absolute Dir | Apache Hadoop installation directory |
+| **[HADOOP_EC7]** | HADOOP_LOG_DIR | Absolute Dir | Apache Hadoop logs directory |
 | **[HADOOP_EH1]** | HADOOP_HDFS_HOME   | Absolute Dir  | Home directory of the Apache Hadoop HDFS component |
 | **[HADOOP_EH2]** | HDFS_DIR    | Relative Dir  | Apache Hadoop HDFS jars |
 | **[HADOOP_EH3]** | HDFS_LIB_JARS_DIR | Relative Dir | Additional Apache Hadoop HDFS jar dependencies |
 | **[HADOOP_EY1]** | HADOOP_YARN_HOME   | Absolute Dir  | Home directory of the Apache Hadoop YARN component |
 | **[HADOOP_EY2]** | YARN_DIR    | Relative Dir  | Apache Hadoop YARN jars |
 | **[HADOOP_EY3]** | YARN_LIB_JARS_DIR | Relative Dir | Additional Apache Hadoop YARN jar dependencies |
+| **[HADOOP_EY4]** | YARN_LOG_DIR    | Absolute Dir  | Apache Hadoop Yarn log directory |
```

Comment:
I don't think it is. This came from IBM's team and I feel it is required as part of HCFS. Further questions should be addressed to @lcham